core/src/core/temporal/activities.py

```
@@ -0,0 +1,534 @@
+            component, stage_activity.resilience_config
+        )
+
+        return _process_success_result(
+            stage_activity,
+            result,
+            attempt_number,
+            start_time,
+            resilience_info,
+        )
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)